### PR TITLE
remove mantle, add lttng image

### PR DESCRIPTION
**Pull request by michaelsevilla**
Signed-off-by: Michael Sevilla mikesevilla3@gmail.com

### CODE REVIEW

File: ceph-lttng/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM ceph-heads/remotes/origin/mds-reqlatency
```
Comment (ivotron) [inline ceph-lttng/Dockerfile:1]: Is this a valid image reference?

File: ceph-lttng/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+===================================================
+Quickstart
+===================================================
+
+Build Ceph with LTTnG and put it in an image:
+
+```bash
+CEPH_SRC=~/ceph ./build
+```
```
Comment (ivotron) [inline ceph-lttng/README.md:12]: Were you able to generate a daemon image with your changes? Looks like we're missing the modifications to `ceph-base/scripts/generate_daemon_image`, were you thinking in having those as a separate changeset?

**Comment by ivotron** [comment]
ignore my review, I just saw you merged. We we're working concurrently

File: ceph-lttng/Dockerfile (hunk 1 above)
Comment (michaelsevilla) [reply ceph-lttng/Dockerfile:1]: The build script uses `dmake` to build an image from the remotes/origin/mds-reqlatency branch. After `dmake`, this Dockerfile builds from that image, which is called ceph-heads/remotes/origin/mds-reqlatency.

File: ceph-lttng/README.md (hunk 2 above)
Comment (michaelsevilla) [reply ceph-lttng/README.md:12]: Yes, it works (although I forgot to add a test to Travis CI). The flow is a little different from your other image builders but I basically add `lttng-tools` to the image by building another image on top of the image produced by `cephbuilder`.

Comment (ivotron) [reply ceph-lttng/README.md:12]: Oh, got it. That makes more sense: you don't have to mess with the daemon script and leave the customization to whoever writes the builder.

**Pull request merged**